packages/components/src/Actions/ActionSplitDropdown/ActionSplitDropdown.component.js

```diff
@@ -87,7 +87,7 @@ ActionSplitDropdown.propTypes = {
 
 ActionSplitDropdown.defaultProps = {
 	items: [],
-	t: getDefaultTranslate,
+	t: getI18n().t.bind(getI18n()),
```

Comment:
Is it possible to avoid to write this bind() everywhere ?
For example in `translate.js`, the getDefaultTranslate is not used anymore with your changes, you can replace it with `return getI18n().t.bind(getI18n())` so in components we can white the defaultProps `t: getDefaultTranslate()`

(same for all components)